README.md

```diff
@@ -100,7 +118,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema formats the columns of a table and how it relates to other elements. In the case of people and wishes you would say that people one wishes (multiple) by saying that the class Wishes belongs_to people and that the class People has_many wishes
 ```
```

Comment:
:+1:

We were hoping for some discussion of primary and foreign keys here.  Including how they might be represented in the tables.